release-notes/opensearch-neural-search.release-notes-2.10.0.0.md

```diff
@@ -0,0 +1,11 @@
+## Version 2.10.0.0 Release Notes
+
+Compatible with OpenSearch 2.10.0
+
+### Features
+* Added Score Normalization and Combination feature ([#241](https://github.com/opensearch-project/neural-search/pull/241/))
```

Comment:
lets change to "Improved Hybrid Search relevancy by Score Normalization and Combination" ?

Reply:
This sounds good. Let me fix it.

Reply:
Fixed.